.github/workflows/publish.yml

```diff
@@ -0,0 +1,17 @@
+name: Publish Package to npm
+on:
+  release:
+    types:
+      - released
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 20
+          registry-url: https://registry.npmjs.org
+      - run: npm publish --access=public --provenance
```

Comment:
`--access=public` is required for the very first time you publish a package, so people wouldn't constantly accidentally publish their packages publicly instead of using their job's own internal package repository or whatever.

`--provenance` adds a green checkmark to the release on the npm that links back to the github action that did the publishing